lib/RoadizCoreBundle/src/Form/NodeTypeFieldsType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RZ\Roadiz\CoreBundle\Form;
+
+use RZ\Roadiz\CoreBundle\Bag\DecoratedNodeTypes;
+use RZ\Roadiz\CoreBundle\Entity\NodeType;
+use RZ\Roadiz\CoreBundle\Entity\NodeTypeField;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Node types selector form field type.
+ */
+class NodeTypeFieldsType extends AbstractType
+{
+    public function __construct(private readonly DecoratedNodeTypes $nodeTypesBag)
+    {
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'nodeType' => null,
+            'choice_label' => fn (NodeTypeField $field) => $field->getName(),
```

Comment:
Maybe use `getLabel` here.

If you want to override a field's label, it will be easier to see default label here. instead of its technical name